ALUM/ALUM/Models/CurrentUserModel.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  CurrentUserModel.swift
```

Comment:
During the duration of our application we will need some data for example who is the current user (needed for all requests sent) + is it a mentor or a mentee? (@HarshGurnani this will be useful in SessionDetailsPage)